tox.ini

```diff
@@ -3,14 +3,13 @@
 envlist = {py36,py39}-{dj32}
           {py38,py310}-{dj40}
           {py38,py310}-{dj41}
-
-toxworkdir = {homedir}/.tox-django-typed-models
```

Comment:
The `{homedir}` substitution was removed in tox 4: https://tox.wiki/en/latest/upgrading.html#substitutions-removed

I think we should stick with the default `.tox` directory, I don't see why it was changed. I also added the `.gitignore` rule for `.tox`.